#
#

Header header
uint32 steeringMode   # Steering mode 0x123 == Differential 0x124 == Ackermann 0x125 == Mixed
